apps/api/src/app/notification-template/usecases/create-notification-template/create-notification-template.usecase.ts

```diff
@@ -19,12 +19,19 @@ export class CreateNotificationTemplate {
     const contentService = new ContentService();
     const variables = contentService.extractMessageVariables(command.messages);
 
+    const templateCheck = await this.notificationTemplateRepository.findByTriggerIdentifier(
+      command.organizationId,
+      `${slugify(command.name, {
+        lower: true,
+        strict: true,
+      })}`
```

Comment:
Maybe we can create a variable out of it instead of calling the slugify multiple times, just to make sure that the options passed to it are consistent,